compiler/testData/diagnostics/tests/smartCasts/publicVals/var.fir.kt

```diff
@@ -4,10 +4,10 @@ public class X {
     public fun fn(): Int {
         if (x != null)
             // Smartcast is not possible for variable properties
-            return x.<!INAPPLICABLE_CANDIDATE!>length<!>
+            return x.<!UNSAFE_CALL!>length<!>
```

Comment:
These should be `SMARTCAST_IMPOSSIBLE` and I left a TODO there.

Will see/learn what _predictable_ nullability means at https://github.com/JetBrains/kotlin/commit/41ebfd025e6cc02cc5559b1a79d6335dd3831cb7 :) Any advices/suggestions are welcome!